### PR TITLE
Initial JMS support

### PR DESCRIPTION
**Pull request by lburgazzoli**
See #424

### CODE REVIEW

**Comment by lburgazzoli** [comment]
ok to test

File: extensions/sjms/runtime/pom.xml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        <artifactId>camel-quarkus-sjms-parent</artifactId>
+        <version>0.4.1-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>camel-quarkus-sjms</artifactId>
+    <name>Camel Quarkus :: SJMS :: Runtime</name>
+
+    <properties>
+        <firstVersion>0.4.1</firstVersion>
```
Comment (ppalaga) [inline extensions/sjms/runtime/pom.xml]: Adding extensions in a micro release sounds wrong. Should be either 0.5.0 (or maybe 1.0.0.RC1?)

**Comment by lburgazzoli** [comment]
@ppalaga, it has been fixed already

File: extensions/sjms/runtime/src/main/resources/META-INF/quarkus-extension.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+---
+name: "Camel Quarkus SJMS"
+description: "Camel Quqrkus JMS support"
```
Comment (ppalaga) [inline extensions/sjms/runtime/src/main/resources/META-INF/quarkus-extension.yaml]: `Quqrkus` :-D

Comment (lburgazzoli) [reply extensions/sjms/runtime/src/main/resources/META-INF/quarkus-extension.yaml]: This PR has already been merged, you are very welcome to fix findings :)

File: extensions/sjms/runtime/pom.xml (hunk 1 above)
Comment (lburgazzoli) [reply extensions/sjms/runtime/pom.xml]: this has already been fixed

**Pull request merged**